vpr/src/route/rr_graph.cpp

```diff
@@ -332,6 +334,21 @@ void create_rr_graph(const t_graph_type graph_type,
                                                                   router_opts.reorder_rr_graph_nodes_threshold,
                                                                   router_opts.reorder_rr_graph_nodes_seed);
             }
+        } else if (!device_ctx.read_rr_graph_filename.size() && router_opts.FPGAInterchange) {
+            free_rr_graph();
+
+            VTR_LOG("Custom RR_graph generator\n");
+            build_rr_graph_fpga_interchange(graph_type,
+                                            grid,
+                                            segment_inf,
+                                            router_opts.base_cost_type,
+                                            &det_routing_arch->wire_to_rr_ipin_switch,
+                                            router_opts.do_check_rr_graph);
+            if (router_opts.reorder_rr_graph_nodes_algorithm != DONT_REORDER) {
+                mutable_device_ctx.rr_graph_builder.reorder_nodes(router_opts.reorder_rr_graph_nodes_algorithm,
+                                                                  router_opts.reorder_rr_graph_nodes_threshold,
+                                                                  router_opts.reorder_rr_graph_nodes_seed);
+            }
```

Comment:
I think we also need to allow reading in the fpga interchange rr graph which, for what I understand, disabled in the previous `if` condition